federation-2/router-bridge/js-src/plan_worker.ts

```diff
@@ -204,9 +211,16 @@ async function run() {
       }
     } catch (e) {
       logger.warn(`plan_worker: an unknown error occured ${e}\n`);
+      let unexpectedError = {
+        name: "unexpectedError",
```

Comment:
It will be unexpected to us, but it potentially has a [meaningful `name` property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name) on the `e` (Error).  More on this in my next comment